docs/console/tutorials/create-extension.md

```diff
@@ -16,54 +16,47 @@ This guide requires that you have:
 - The Company has a project reachable externally, otherwise create one by referring to this [guide](/console/project-configuration/create-a-project.mdx).
 - The `Microfrontend Composer Toolkit` application created and exposed to view the Backoffice frontend. Follow this section of the [guide](/microfrontend-composer/tutorials/basics.mdx#setup-the-microservices) to do so.
 - The response from the Backoffice endpoint can be embedded within an iframe. See this [link](/development_suite/api-console/api-design/endpoints.md#manage-advanced-endpoint-parameters) for more information.
+- Miactl tool installed consulting the guide on this [link](/cli/miactl/20_setup.md)
 
 
 ## 1. Register Backoffice Extension
 
-Once that all requisites are satisfied, you can register the Backoffice as an extension using the [PUT /api/extensibility/tenants/{tenantId}/extensions API](/console/console-extensibility/registration.md#how-to-register-your-extension):
+Once that all requisites are satisfied, you can register the Backoffice as an extension using the following command:
 
-![registration example](img/registrationExample.png)
-![registration example routes](img/registrationExampleRoutes.png)
-
-**Path Params**
-```json
-{
-  "tenantId": "my-tenant-id"
-}
+```sh
+miactl extensions apply --endpoint https://<my-console-domain>/ --company-id my-tenant-id -f ./<relative-path>/my-extension.json
 ```
 
-**Body Params**
+where in the file `my-extension.json` there are the info to register:
+
 ```json
 {
-  "contexts": ["project"],
+  "name": "Integrated Backoffice",
```

Comment:
same here, let's use collapsed panels

Reply:
The collapsed panels reduces the user attention on their content because it seems optional, instead here it is an important part of tutorial

Reply:
I understand but they are still a friction when reading the tutorial in its entirety